README.md

```diff
@@ -205,15 +204,9 @@ address for the publicly facing network interface on the host.
 
 #### Building docs
 
-First, navigate to `/src/{plugin}/docs` in the container and do the following:
-```
-pip3 install -r ../doc_requirements.txt
-make clean && make diagrams && make html
-python3 -m http.server ${DOCS_PORT}
-```
-
-Then, visit `0.0.0.0:${DOCS_PORT}` in a web browser to preview the documentation.
+Install the docs building utility from https://github.com/pulp/pulp-docs.
 
+Then, run `make docs` and `make servedocs`.
```

Comment:
This is just for building the docs? What I need to do to reach those docs?

Reply:
If you run make docs, it runs: pulp-docs build and if you run make servedocs, it runs: pulp-docs serve. The URL is accessible from the output logs. You can run these commands outside of the oci-env, like so:
```
$ make docs
```
```
pulp-docs build
Building: mkdocs build --config-file /home/lmjachky/Workplace/pulp/pulp_container/venv3/lib64/python3.12/site-packages/pulp_docs/data/mkdocs.yml --site-dir /home/lmjachky/Workplace/pulp/oci_env/site
....
```
```
$ make servedocs
```
```
pulp-docs serve
Running: mkdocs serve --config-file /home/lmjachky/Workplace/pulp/pulp_container/venv3/lib64/python3.12/site-packages/pulp_docs/data/mkdocs.yml
...
WARNING -  [pulp-docs] Disabled features: ['blog']
    Blog rendering is disabled for faster reloads.
    To enable it, use: -a, --enable-all
    
INFO    -  Documentation built in 17.25 seconds
INFO    -  [10:11:10] Watching paths for changes: 'staging_docs'
INFO    -  [10:11:10] Serving on http://127.0.0.1:8000/
```